t/t1000-kvs-basic.t

```diff
@@ -359,54 +359,54 @@ test_expect_success 'kvs: symlink: kvs_copy does not follow symlinks (bottom)' '
 '
 
 test_expect_success 'kvs: get_symlinkat works after symlink unlinked' '
-	${KVSBASIC} unlink $TEST &&
+	flux kvs unlink -Rf $TEST &&
 	flux kvs link $TEST.a.b.X $TEST.a.b.link &&
 	ROOTREF=$(${KVSBASIC} get-treeobj .) &&
-	${KVSBASIC} unlink $TEST &&
+	flux kvs unlink -R $TEST &&
```

Comment:
I think we keep this one at ```-R``` b/c we want the unlink to pass given the path created above it.